core/templates/components/common-layout-directives/navigation-bars/side-navigation-bar.component.html

```diff
@@ -88,66 +88,73 @@
   </div>
 
   <ul>
-    <li ng-class="{'active': $ctrl.currentUrl === '/community-library'}">
+    <li ng-class="{'active': currentUrl === '/community-library'}">
       <a href="/community-library" class="protractor-mobile-test-library-link">
         <i class="material-icons oppia-sidebar-menu-icon">&#xE88A;</i>
+        <!--TODO(orthodoxparadox): Fix translate-->
```

Comment:
Could you please take a look at your PR" https://github.com/oppia/oppia/pull/9790 and apply all the comments that are added there in this PR too? Looks like there are similar things that need to be modified.